addon/content/es-modules/thunderbird-compat.js

```diff
@@ -0,0 +1,219 @@
+    description: "Options - description for if to show friendly dates.",
+  },
+
+  "options.tweak_chrome_title": {
+    message: "Tweak chrome fonts",
+    description: "Options - title for tweaking chrome fonts.",
+  },
+
+  "options.tweak_chrome_desc": {
+    message:
+      "Scale down the font size for the conversation chrome (recommended, requires restart).",
+    description: "Options - description for tweaking chrome fonts.",
+  },
+
+  "options.tweak_bodies_title": {
+    message: "Tweak body fonts",
+    description: "Options - title for tweaking message body fonts.",
+  },
+
+  "options.tweak_bodies_desc": {
+    message:
+      "Tweak the fonts in the message bodies to make them consistent with the outer font (recommended, requires restart).",
+    description: "Options - description for tweaking message body fonts.",
+  },
+
+  "options.operate_on_conversations_title": {
+    message: "Archive/delete behavior",
+    description: "Options - title for archive/delete behavior.",
+  },
+
+  "options.operate_on_conversations_desc": {
+    message:
+      "Toolbar buttons should archive and delete the entire conversation, not just selected messages.",
+    description: "Options - description for archive/delete behavior.",
+  },
+
+  "options.extra_attachments_title": {
+    message: "I love attachments",
+    description: "Options - title for attachment settings.",
+  },
+
+  "options.extra_attachments_desc": {
+    message:
+      "In some corner cases, Thunderbird will fail to recognize some attachments. This option allows you to workaround the Thunderbird bug, and makes sure we try to display as many attachments as we can. Use in combination with the Show All Body Parts addon.",
+    description: "Options - description for attachment settings.",
+  },
+
+  "options.compose_in_tab_title": {
+    message: "Hidden composition window",
+    description: "Options - title for hidden compose window enabling.",
+  },
+
+  "options.compose_in_tab_desc": {
+    message:
+      "Ctrl-Shift-N opens the hidden, unsupported composition window in a new tab",
+    description: "Options - description for hidden compose window enabling.",
+  },
+
+  "options.debugging_title": {
+    message: "Debugging",
+    description: "Options - title for debugging setting.",
+  },
+
+  "options.debugging_desc": {
+    message:
+      "Enable debug output in the system console and the error console. This will generate a lot of messages.",
+    description: "Options - description for debugging setting.",
+  },
+
+  "options.monospaced_senders_title": {
+    message: "Display in monospaced",
+    description: "Options - title for monospaced sender setting.",
+  },
+
+  "options.monospaced_senders_desc": {
+    message:
+      "Messages from those addresses/mailing-lists should be displayed in monospaced fonts (separate using commas).",
+    description: "Options - description for monospaced sender setting.",
+  },
+};
+const DEFAULT_PREFS = {
```

Comment:
Where are the default prefs located? The trouble is for es modules, one must use the export keyword (or store something as a global) and the export keyword breaks normal scripts. Maybe it can be put in a try-catch statement...? But I think this is forbidden since exports need to be statically analyzable.

Reply:
I found `prefs.js`. I'm a little confused about what's going on in that file...should I just import the `Prefs` object and instantiate a new copy? It looks like there's automatic migration stuff that goes on. Did we want that triggered in the dev version?